src/ComposerHandler.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace JeroenG\Packager;
+
+use Closure;
+use RuntimeException;
+use Symfony\Component\Finder\Exception\DirectoryNotFoundException;
+
+trait ComposerHandler
+{
+    use ProcessRunner;
+
+    protected function removeComposerRepository($name)
+    {
+        return self::modifyComposerJson(function (array $composer) use ($name){
+            unset($composer['repositories'][$name]);
+            return $composer;
+        }, base_path());
+    }
+
+    /**
+     * Determines the path to Composer executable
+     * @todo Might not work on Windows
```

Comment:
I have a Windows machine, so I will test this.

Reply:
Cool. I've removed the @todo